modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -131,6 +131,16 @@ func (a TransferAuthorization) ValidateBasic() error {
 			}
 			found[allocation.AllowList[i]] = true
 		}
+
+		if len(allocation.AllowedPacketData) > 0 && allocation.AllowedPacketData[0] != AllowAllPacketDataKeys {
+			jsonObject := make(map[string]interface{})
+			for _, elem := range allocation.AllowedPacketData {
+				err := json.Unmarshal([]byte(elem), &jsonObject)
+				if err != nil {
+					return errorsmod.Wrapf(ErrInvalidAuthorization, "allowed packet data contains a non JSON-encoded string: %s", elem)
```

Comment:
Why do we require them to be JSON encoded? Can we remove this requirement please and just use any string?

Reply:
Sure, I can remove it.

Reply:
Ok, I changed it [here](https://github.com/cosmos/ibc-go/pull/6268/commits/34735f0ad5a4decb89f598ed46400c9326202470). If during the PR review we decide to go back to enforcing JSON, we can just revert that commit.